marketpulse/main/static/js/fxosprice_view.js

```diff
@@ -9,5 +9,6 @@ $(document).ready(function() {
     $(document).on('click', '.cancel', function () {
         $(this).parents('.delete-contribution').find('.ask').show();
         $(this).closest('.delete-confirm').hide();
+        return false;
```

Comment:
We need to this to prevent the page from scrolling on top, due to the `href="#"`. It's equivalent to `event.preventDefault()`.